pkgs/tools/audio/dl-librescore/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, buildNpmPackage
+, fetchFromGitHub
+, python3
+, darwin
+}:
+
+buildNpmPackage rec {
+  pname = "dl-librescore";
+  version = "0.34.47";
+
+  src = fetchFromGitHub {
+    owner = "LibreScore";
+    repo = "dl-librescore";
+    rev = "v${version}";
+    hash = "sha256-yXreyQiIKmZEw2HcpnCW4TxCTHzdq+KuPSlFPFZy2oU=";
+  };
+
+  npmDepsHash = "sha256-qKu7xViApKg/4EubS4tsZEtNoW62rpC4e6xmBugSkek=";
+
+  postPatch = ''
+    # https://github.com/LibreScore/dl-librescore/pull/32
```

Comment:
Can you hoist this comment above the `postPatch` (so it doesn't affect the drv hash) and mark it with `TODO: can be removed [insert condition here]` so it's clear when this `postPatch` can be removed?

Thank you again for this!

Reply:
I don't like the former because it makes things confusing if multiple things are done in `postPatch`.

Reply:
That's fine then, it's more a personal preference so I probably shouldn't have asked for it anyway. Feel free to leave it inside the `postPatch`, but it would be good to know when it can be removed